portfolio/src/main/webapp/script.js

```diff
@@ -31,13 +31,21 @@ function nextImage() {
 /**
  * Fetches data and adds to html
  */
-function getHelloNameData() {
-  fetch('/data').then(response => response.json()).then((hello) => {
+function getComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
       const msgContainer = document.getElementById('hello-container');
+      msgContainer.innerHTML = "";
 
-      hello.forEach(function(message) {
-        msgContainer.innerHTML += message;
+      comments.forEach(function(message) {
+        msgContainer.innerHTML += message + " ";
       });
       
     });
 }
+
+/**
+ * Deletes data
+ */
+function deleteComments() {
+  fetch('/delete-data', {method: 'POST', body: ""}).then(getComments());
+}
```

Comment:
Question: Does this work yet?

Reply:
If not, maybe we can move it to another PR?

Reply:
It does work, just forgot to add the new file! My bad